core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -54,7 +54,7 @@ import org.apache.spark.util.{AccumulatorV2, ThreadUtils, Utils}
 private[spark] class TaskSchedulerImpl private[scheduler](
     val sc: SparkContext,
     val maxTaskFailures: Int,
-    blacklistTrackerOpt: Option[BlacklistTracker],
+    val blacklistTrackerOpt: Option[BlacklistTracker],
```

Comment:
doh, actually this is used in tests.  I override `createTaskSetManager` to use my manual clock.  Another alternative is to allow the clock to get passed in to the `TaskSCheudlerImpl` constructor, but that ends up being a bigger code change.

I'll make it `private[scheduler] val`, slightly tighter than the implicit `private[spark]` it would be otherwise.